tests/setup_cfg_fmt_test.py

```diff
@@ -274,7 +278,9 @@ def test_license_does_not_match_directories(tmpdir):
 
 
 def test_rewrite_sets_license_type_and_classifier(tmpdir):
-    with open('LICENSE') as f:
+    here = os.path.dirname(__file__)
```

Comment:
Added this to ensure the test runs independent of the CWD.

Reply:
where else would the tests run from?

Reply:
From within the tests folder itself? If you use an IDE it will set the CWD to the ``tests`` folder and then things break.